spec/secrets.yml

```
@@ -190,6 +190,12 @@ components:
             $ref: 'openapi.yml#/components/responses/InadequatePrivileges'
           "404":
             $ref: 'openapi.yml#/components/responses/ResourcesNotFound'
+          "406":
+            description: The response was unable to be encoded in JSON
```

```suggestion
            description: Issue encoding secret into JSON format
```
maybe keep this the same as the server message?